src/main/webapp/app/localci/build-agents/build-agent-summary/build-agent-summary.component.ts

```
@@ -76,9 +76,9 @@ export class BuildAgentSummaryComponent implements OnInit, OnDestroy {
     }
 
     cancelAllBuildJobs(buildAgentName: string) {
-        const buildAgent = this.buildAgents.find((agent) => agent.name === buildAgentName);
-        if (buildAgent && buildAgent.name) {
-            this.buildQueueService.cancelAllRunningBuildJobsForAgent(buildAgent.name).subscribe();
+        const buildAgent = this.buildAgents.find((agent) => agent.buildAgent?.name === buildAgentName);
+        if (buildAgent && buildAgent.buildAgent?.name) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)